apps/design/backend/src/worker/worker.ts

```diff
@@ -1,45 +1,46 @@
 import { extractErrorMessage, sleep } from '@votingworks/basics';
 
-import { BackgroundTask, Store } from '../store';
+import { Workspace } from '../workspace';
+import { processBackgroundTask } from './tasks';
 
-async function processBackgroundTask(
-  _store: Store, // eslint-disable-line @typescript-eslint/no-unused-vars
-  _task: BackgroundTask // eslint-disable-line @typescript-eslint/no-unused-vars
-): Promise<void> {
-  // Simulate a long-running operation
-  // TODO(arsalan): Implement actual processing logic
-  await sleep(3000);
-}
+export async function processNextBackgroundTaskIfAny({
```

Comment:
Easier to review this commit using https://github.com/votingworks/vxsuite/pull/4462/commits/50816f451894720fb256c5c539e1d51e8517d11c?w=1

Doing some rearranging to facilitate testing. Rather than having app tests additionally start up a background worker, app tests can just call `processNextBackgroundTaskIfAny` as needed. You'll see this in the next commit